elisp-bytecode.texi

```diff
@@ -536,6 +544,35 @@ Make a binding recording buffer, point, and mark.
 @item @verb{|138|}
 @tab @code{byte-save-excursion}
 @tab Make a binding recording buffer, point, and mark.
+@item @verb{|169|}
+@tab
+@tab Unused
+@item @verb{|170|}
+@tab
+@tab Unused
+@item @verb{|171|}
+@tab
+@tab Unused
+@item @verb{|172|}
+@tab
+@tab Unused
+@item @verb{|173|}
+@tab
+@tab Unused
+@item @verb{|174|}
+@tab
+@tab Unused
+@item @verb{|175|}
+@tab byte-listN
+@tab
+@item @verb{|176|}
+@tab byte-concatN)
+@item @verb{|177|}
+@tab byte-insertN
+@item @verb{|178|}
+@tab byte-stack-set
+@item @verb{|179|}
+@tab byte-stack-set2
```

Comment:
I think it would be consistent to call this `byte-stack-set` (no 2), but I haven't verified it disassembles as such.

Reply:
Just confirmed, it disassembles as `(byte-stack-set . 279)`

Reply:
Ok. But looking at bytecode.c one has a value operand and the other doesn't. That needs to be expressed somehow. Suggest something.

Reply:
I think it's just a matter of whether the operand is one or  two bytes? My suggestion would be to list `byte-stack-set` in the table, and have a footnote explaining that there are two opcodes and that the shortest valid encoding is canonically used (I don't think that's a requirement).

Reply:
Well, the opcode value is 178 or 179. And if you look at bytecode.c  it is the opcode which signals whether the operand 8 or 16 bits. 

My take is to follow bytecode.c and bytecomp.el which both consider these distinct. If various disassembly routines treat this as one thing, that is a user interface convenience. Underneath they are two different instructions which do almost the same thing.

Reply:
You've convinced me! Yes, it's better to document them that way.